composer.json

```diff
@@ -11,9 +11,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3",
-        "symfony/framework-bundle": "2.*",
-        "symfony/symfony": "2.*"
+        "php": ">=5.5.9",
```

Comment:
Required
    PHP needs to be a minimum version of PHP 5.5.9
http://symfony.com/doc/current/reference/requirements.html